### PR TITLE
Split [pypi] tests

### PR DESCRIPTION
**Pull request by PyvesB**
See https://github.com/badges/shields/pull/3701#issuecomment-511017547. Quite a bit of  splitting to do here.

### CODE REVIEW

File: services/pypi/pypi-djversions.service.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
 const PypiBase = require('./pypi-base')
 const { sortDjangoVersions, parseClassifiers } = require('./pypi-helpers')
 
-module.exports = class PypiDjangoVersions extends PypiBase {
+module.exports = class PypiDjVersions extends PypiBase {
```
Comment (PyvesB) [inline services/pypi/pypi-djversions.service.js]: To be consistent with the route path and avoid having to override the `id` in the `ServiceTester`.

Comment (calebcartwright) [reply services/pypi/pypi-djversions.service.js]: I think I'd have had a slight preference for making the filenames more explicit to match the class names, but this is fine too

Comment (paulmelnikow) [reply services/pypi/pypi-djversions.service.js]: For future reference, I don't think the class names need to match the route path, even for `createServiceTester()`. We've quite a few that don't.

Comment (PyvesB) [reply services/pypi/pypi-djversions.service.js]: I think I'll go with @calebcartwright suggestion and change the filenames instead. But I've got a slight preference for sleep right now, so will do that sometime in the morning! :grin:

**Pull request merged**